projects/xbanking/index.js

```diff
@@ -0,0 +1,23 @@
+const sdk = require('@defillama/sdk');
+const ADDRESSES = require('../helper/coreAssets.json')
+const ETHEREUM_CONTRACT = "0xC4356aF40cc379b15925Fc8C21e52c00F474e8e9";
+
+
+async function tvl(_, _1, _2, { api }) {
+  const bal = await api.call({
+    abi: 'function getTvl() external view returns (uint256)',

```

Comment:
I have no clue what is going on here/why the number returned here should be taken as Tvl.

Can you give a token breakdown/where the tokens are locked?

Reply:
how I can test this code?

Reply:
Try running "node test.js projects/xbanking"

Reply:
where I need to run that?